test/stripe/radar/early_fraud_warning_test.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require ::File.expand_path("../../test_helper", __FILE__)
+
+module Stripe
+  module Radar
+    class EarlyFraudWarningTest < Test::Unit::TestCase
+      should "be listable" do
+        issfrs = Stripe::Radar::EarlyFraudWarning.list
```

Comment:
IMO this variable is a little weird for Ruby — should we just call it `warning` or something?

Reply:
Oops, yeah, definitely. Fixed.